### ISSUE TITLE
[BUG] When enabled maintenance banner does not appear on /sign-up and /pxp pages

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8344]
### Description

When the maintenance banner is enabled, it does not appear on the /sign-up and /pxp flows

### Steps to reproduce

1. Enable the maintenance banner on a lower env
2. Once enabled, go to the /sign-up and /pxp pages
3. Notice no maintenance banner is present

### Expected behavior

When enabled, the maintenance banner should appear for all pages

### Additional context

Discovered while fixing #8342

**Comment by arinkulshi-skylight** [comment]
This should be an easy fix but please verify it:
navigation.yml file:      
```
 - page: Backdate a test
            url: /using-simplereport/conduct-and-submit-tests/backdate-a-test/ 
```
change it to
        -   ```
        - page: Backdate a test
            url: /using-simplereport/report-test-results/backdate-a-test/
```

**Comment by arinkulshi-skylight** [comment]
@jayna-SkylightDigital @DanielSass a quick question on this ticket. This ticket mentions adding the maintenance banner to the sign-up and and /pxp pages. Should we also add the banner to the self-registration page? `/register/:registrationLink`

**Comment by jayna-SkylightDigital** [comment]
@arinkulshi-skylight Good catch! Please add the maintenance banner to the self-registration page as well. Thanks!

**Comment by arinkulshi-skylight** [comment]
Thanks Jayna I will add that in!
